apps/public/public_test.go

```diff
@@ -43,16 +44,16 @@ func fakeBrowserOpenURL(authURL string) error {
 	if m := q.Get("code_challenge_method"); m != "S256" {
 		return fmt.Errorf("unexpected code_challenge_method '%s'", m)
 	}
-	if q.Get("prompt") == "" {
-		return errors.New("missing query param 'prompt")
-	}
+	// if q.Get("prompt") == "" {
+	// 	return errors.New("missing query param 'prompt")
+	// }
 	state := q.Get("state")
 	if state == "" {
 		return errors.New("missing query param 'state'")
 	}
 	redirect := q.Get("redirect_uri")
 	if redirect == "" {
-		return errors.New("missing query param 'redirect_uri'")
+		return errors.New(" 'redirect_uri'")
```

Comment:
Providing a bit more detail in the error message helps with debugging just saying 'redirect_uri' is too short. Including what exactly is wrong (e.g., missing, invalid, mismatched) would be more helpful.
```suggestion
		return errors.New("missing redirect param 'redirect_uri'")
```
revert to original